mne/io/egi/source/data.py

```diff
@@ -0,0 +1,82 @@
+"""Read the data."""
+
+
+def read_mff_data(filepath, indtype, startind, lastind, hdr):
+    """Function for load the data in a list.
+
+    Parameters:
+    filepath = str
+    indtype = "sample" or "epoch"
+    startind = the start index (int)
+    lastind = the last index (int)
+    hdr = the header array from read_mff_header
+    """
+    import numpy as np
+    import os
+    from .general import _bls2blns, _read_signaln, _get_gains
```

Comment:
don't nest these

hopefully `general` does not create a circular import. If it does, some helper function probably needs to be moved